.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
-- [ ] closes #xxxx
-- [ ] tests added / passed
-- [ ] passes `black pandas`
-- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
-- [ ] whatsnew entry
+* [ ] closes [#38696](https://github.com/pandas-dev/pandas/issues/38694)
+* [ ] tests added / passed
+* [ ] passes `pre-commit run --from-ref=upstream/master --to-ref=HEAD --all-files`
```

Comment:
can you put this one in the `code standards` section of the contributing guide in place of
```
./ci/code_checks.sh
```
and remove it from here?

Reply:
So the PULL_REQUEST_TEMPLATE.md file should contain this  👇
- [ ] closes [#38696](https://github.com/pandas-dev/pandas/issues/38694)
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

and contributing.rst file should contain this 👇 replacing  👉 `./ci/code_checks.sh`

- [ ] closes [#38696](https://github.com/pandas-dev/pandas/issues/38694)
- [ ] tests added / passed
- [ ] passes `pre-commit run --from-ref=upstream/master --to-ref=HEAD --all-files`
- [ ] whatsnew entry